.travis.yml

```diff
@@ -2,16 +2,14 @@ language: node_js
 node_js:
   - "4.4"
 before_install:
-  - npm install -g grunt-cli bower
+  - npm install -g --production yarn
 install:
-  - npm install
+  - yarn install
```

Comment:
It could. I think `yarn install` is more obvious to the reader. I can change if you prefer.